views/js/qtiCreator/tpl/outcomeEditor/listing.tpl

```diff
@@ -18,6 +18,16 @@
             <div class="tooltip-content">{{__ "A human interpretation of the variable's value."}}</div>
             <input name="interpretation" value="{{interpretation}}" type="text">
         </div>
+        <div class="panel longinterpretation">
+            <label for="longInterpretation" class="has-icon">{{__ "Long interpretation"}}</label>
+            <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+            <div class="tooltip-content">{{__ "An optional link to an extended interpretation of the outcome variable."}}</div>
+            <input name="longInterpretation"
+                   placeholder="https://www.tao.org/rules.pdf"
```

Comment:
Wouldn't that be better to have more generic url example? Something like `https://example.com/doc.pdf`.
Initially, I thought ist some kind of technical reference. Also, can we do something about such cases as below? 
![image](https://user-images.githubusercontent.com/5583430/77147483-9de22d00-6a8d-11ea-80d4-64f7a5612c2d.png)

Reply:
agreed